dist/operators/s2r.d.ts

```diff
@@ -42,5 +42,10 @@ export declare class CSPARQLWindow {
     add(e: Quad, timestamp: number): void;
     compute_report(w: WindowInstance, content: QuadContainer, timestamp: number): boolean;
     scope(t_e: number): void;
-    subscribe(output: 'RStream' | 'IStream' | 'DStream', call_back: (data: QuadContainer) => void): void;
+    subscribe(output: 'RStream' | 'IStream' | 'DStream', call_back: (data: container_with_bounds) => void): void;
 }
+export type container_with_bounds = {
```

Comment:
Why is container_with_bounds not camel case like the other types?

Reply:
I'm also not sure why you need it? The QuadContainer already has the timestamp the window triggered,i.e. last_time_stamp_changed.